README.md

```diff
@@ -44,8 +44,10 @@ docker run -d \
   --net="host" \
   --pid="host" \
   -v "/:/host:ro,rslave" \
+  -v  "/var/lib/node_exporter/textfile_collector:/var/lib/node_exporter/textfile_collector"
   quay.io/prometheus/node-exporter:latest \
-  --path.rootfs=/host
+  --path.rootfs=/host --path.udev.data=/host/run/udev/data --path.procfs=/host/proc --path.sysfs=/host/sys
```

Comment:
This shouldn't need to be changed.

Reply:
Does `--path.rootfs=/host` imply all the following flags?

Reply:
The change in #2941 is not obvious and `diskstats` is enabled by default

Reply:
> The --path.rootfs flag is supposed to adjust all path defaults so only the one flag is needed.
Would you mind pointing out an instance of that happening in the code for `/proc` or `/sys`?

Reply:
For example, the `os_release.go` uses `rootfsPath` to build the path.

Reply:
Ah for `/proc` and `/sys` there is no explicitness in the code because of this I think https://github.com/prometheus/node_exporter/pull/912#discussion_r191388817

So its possible that the udev flag has been added but the `--path.rootfs` is not used. I'll attempt a PR.

Reply:
Correct, but those two paths (`/proc` and `/sys`) don't need redirect by default because the node_exporter is executed in the host PID namespace in order to access host metrics.

IIRC, we used to have examples of adjusting `--path.procfs` with a volume mount. But it turned out we still needed host PID anyway. So those recommendations were dropped. Now we only need to use `--path.rootfs`.

Reply:
Either the udev flag is better documented and explained or #3216